### PR TITLE
Add script to create Github release based on beachball changelog

### PR DESCRIPTION
**Pull request by jasonnutter**
Adds a script to programmatically create Github releases based on the latest release for each library. Used to create the most recent releases: https://github.com/AzureAD/microsoft-authentication-library-for-js/releases

### CODE REVIEW

File: .github/create-releases.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    const release = {
+        ...repoMeta,
+        tag_name,
+        name: `${name} v${version}`,
+        prerelease: !!prerelease,
+        body: body.join("\n")
+    };
+
+    // Gets latest releases and checks if there is a matching releases for the given library version
+    // TODO: improve check to handle release not being on the first page
```
Comment (pkanher617) [inline .github/create-releases.js:94]: what does this mean?

File: .github/create-releases.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    const release = {
+        ...repoMeta,
+        tag_name,
+        name: `${name} v${version}`,
+        prerelease: !!prerelease,
+        body: body.join("\n")
+    };
+
+    // Gets latest releases and checks if there is a matching releases for the given library version
```
Comment (pkanher617) [inline .github/create-releases.js:93]: This is only checking releases that we have created entries for in Github right? Should we also add a check to see if the release is on npm?

File: .github/create-releases.js (hunk 1 above)
Comment (jasonnutter) [reply .github/create-releases.js:94]: The releases API is paginated, and right now were only looking for an existing release on the first page of results.

File: .github/create-releases.js (hunk 2 above)
Comment (jasonnutter) [reply .github/create-releases.js:93]: We could, although going forward there should be a 1:1 between the two.

**Review by tnorling — APPROVED** [review]

File: .github/create-releases.js (hunk 2 above)
Comment (jasonnutter) [reply .github/create-releases.js:93]: I'll look at checking npm incrementally, as npm will throw an error if you try to republish the same version.

**Review by jmckennon — APPROVED** [review]

**Pull request merged**